src/components/EditableCard.tsx

```diff
@@ -55,50 +64,66 @@ class EditableCard extends Component<IEditableCardProps> {
   }
 
   private get deleteButton () {
-    const { isGuarded, title, onDelete, isLoading } = this.props
-      , classNameSuffix = this.props.classNameSuffix || kebabCase(title);
-
+    const {
+      classNameSuffix,
+      disableDeleteTooltip,
+      disableDelete,
+      isGuarded,
+      isLoading,
+      onDelete,
+      title,
+    } = this.props,
+    className = getBtnClassName('delete', classNameSuffix, title);
     if (!onDelete) { return; }
 
     return (
-      <GuardedButton
-        className={`btn-delete btn-delete-${classNameSuffix}`}
-        confirm={true}
-        disabled={isLoading || this.isDeleting.isTrue}
-        icon='delete'
-        isGuarded={isGuarded}
-        onClick={this.handleDelete}
-        size='small'
-        type='danger'
-      >
-        Delete
-      </GuardedButton>
+      <Tooltip title={disableDelete ? disableDeleteTooltip : ''}>
+        <span>
+          <GuardedButton
+            className={className}
+            confirm={true}
+            disabled={disableDelete || isLoading || this.isDeleting.isTrue}
+            icon='delete'
+            isGuarded={isGuarded}
+            onClick={this.handleDelete}
+            size='small'
+            type='danger'
+          >
+            Delete
+          </GuardedButton>
+        </span>
+      </Tooltip>
     );
   }
 
   private get editButton () {
-    const { isLoading, title, isGuarded } = this.props
+    const { isLoading, title, isGuarded, disableEdit, disableEditTooltip } = this.props
       , classNameSuffix = this.props.classNameSuffix || kebabCase(title);
 
     return (
-      <GuardedButton
-        className={`btn-edit btn-edit-${classNameSuffix}`}
-        disabled={isLoading || this.isEditing.isTrue || this.isDeleting.isTrue}
-        icon='edit'
-        isGuarded={isGuarded}
-        onClick={this.isEditing.setTrue}
-        size='small'
-        type='primary'
-      >
-        Edit
-      </GuardedButton>
+      <Tooltip title={disableEdit ? disableEditTooltip : ''}>
+        <span>
+          <GuardedButton
+            className={`btn-edit btn-edit-${classNameSuffix}`}
+            disabled={isLoading || this.isEditing.isTrue || this.isDeleting.isTrue || disableEdit}
+            icon='edit'
+            isGuarded={isGuarded}
+            onClick={this.isEditing.setTrue}
+            size='small'
+            type='primary'
+          >
+            Edit
+          </GuardedButton>
+        </span>
+      </Tooltip>
     );
   }
 
   private buttons () {
     return (
       <ButtonToolbar noSpacing>
         {this.deleteButton}
+        <div style={{ display: 'inline-block', width: '10px' }}></div>
```

Comment:
Can we try using an `antd` component for this? i.e.

import { Space, Button } from 'antd';

function MyComponent() {
  return (
    <Space size={10}>
      {this.deleteButton}
      {this.editButton}
    </Space>
  );
}